rclpy/rclpy/impl/rmw_implementation_tools.py

```diff
@@ -85,23 +83,8 @@ def import_rmw_implementation():
     if __rmw_implementation_module is not None:
         return __rmw_implementation_module
     available_implementations = get_rmw_implementations()
-    if __selected_rmw_implementation is None:
-        logger = logging.getLogger('rclpy')
-        # prefer FastRTPS, otherwise first in alphabetical order
-        # the same logic is implemented in
-        # rmw_implementation_cmake/cmake/get_default_rmw_implementation.cmake
-        default_impl = 'rmw_fastrtps_cpp'
-        if default_impl in available_implementations:
-            select_rmw_implementation(default_impl)
-        elif available_implementations:
-            select_rmw_implementation(available_implementations[0])  # select the first one
-        else:
-            raise NoImplementationAvailableException()
-        logger.debug("Implicitly selecting the '{0}' rmw implementation."
-                     .format(__selected_rmw_implementation))
-    module_name = '._rclpy__{rmw_implementation}'.format(
-        rmw_implementation=__selected_rmw_implementation,
-    )
+    __selected_rmw_implementation = 'rmw_implementation'
```

Comment:
I guess this is temporary? (pr is still "in progress") You will need to do something with the `select_rmw_implementation()` function. I don't know if you can preserve the functionality here that has to do with selecting the rmw implementation after launching Python but before running `rclpy.init` without also requiring the meta rmw implementation you just made. It might be better to continue to build a Python module for each rmw implementation (since it doesn't really affect user code anyways).